### DISCUSSION TITLE
cp: - r not specified; omitting directory '/docker-entrypoint-mount.sh'

### DISCUSSION THREAD

**Discussion by hy-1998** [langgenius/dify#11471] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

After DIFY is deployed and started through Docker Compose, check the nginx logs in the Docker container and see cp: - r not specified; omitting directory '/docker-entrypoint-mount.sh'。 May I ask how to solve this

**Comment by crazywoola** [comment]
Clone the repo, run the command in the `docker/` dir.
